source/plugin/basics/commands.rst

```diff
@@ -340,14 +344,35 @@ Use the ``setChildren()`` method of the parent command builder to apply the chil
 
     If a ``CommandExecutor`` was set for the parent command, it is used as a fallback if the arguments do not match one of the child command aliases.
     Setting an executor is not required.
-    
+
+Command Results
+===============
+
```

Comment:
Maybe a short explanation how the result is used, with link to minecraftwiki commandblock page